src/__tests__/components/ButtonPanel.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from 'react-test-renderer';
+import ButtonPanel from '../../components/ButtonPanel';
+
+test('renders Display and ButtonPanel components', () => {
+  const snap = renderer.create(<ButtonPanel />).toJSON();
```

Comment:
- [ ] [REQUIRED] This test unit returns an error when I run it. It will be nice if you fix it.
![1](https://user-images.githubusercontent.com/58552748/139708425-b770b6fd-110b-4bf0-9352-4321ca99ce06.gif)